build.assets/Dockerfile-grpcbox

```diff
@@ -8,6 +8,13 @@ RUN apt-get update && \
     && \
   rm -rf /var/lib/apt/lists/*
 
+# Add the dev user.
+ARG UID
+ARG GID
+RUN groupadd dev --gid=$GID -o && \
```

Comment:
This seems like a poor solution to me, as it makes the image depend on the local UID/GID. It doesn't seem like a good first image layer either.

Ideally this Dockerfile shouldn't care about the host user running it. I think we should look for alternative solutions.